ee/app/engagement-dashboard/client/components/UsersTab/BusiestChatTimesSection.js

```diff
@@ -127,7 +127,7 @@ const ContentForDays = ({ displacement, onPreviousDateClick, onNextDateClick })
 	const { value: data } = useEndpointData('engagement-dashboard/users/chat-busier/weekly-data', params);
 	const values = useMemo(() => (data ? data.month.map(({ users, day, month, year }) => ({
 		users,
-		day: String(moment.utc([year, month - 1, day, 0, 0, 0]).valueOf()),
+		day: String(moment({ year, month: month - 1, day }).valueOf()),
```

Comment:
Yeah, I think so 🤔. This way we'll display the data at the same timezone used in the other charts. Otherwise, we'd have to move all the charts timezones to UTC, which I think makes everything more "consistent", BUT it could be confusing for most users, right? Since some dates could be shown 1 day earlier (compared to the dates from their timezones).

But if we're keeping this date at the web browser's timezone, I think we should remove the UTC conversion from `currentDate`, wdyt?